raiden/tests/unit/test_channelstate.py

```diff
@@ -277,6 +278,9 @@ def test_channelstate_update_contract_balance():
 
     assert_partner_state(new_state.our_state, new_state.partner_state, our_model2)
     assert_partner_state(new_state.partner_state, new_state.our_state, partner_model2)
+    # Also make sure that a fee update triggering event is created
+    assert len(iteration.events) == 1
+    assert isinstance(iteration.events[0], TriggerFeeUpdate)
```

Comment:
This could use `search_for_item` to be less specific about order of events

Reply:
Good point. Since I already rebased the other PR on this I will address this from https://github.com/raiden-network/raiden/pull/4802